Update distributed-tracing.md (#7084)
opensearch-project/documentation-website: _observing-your-data/trace/distributed-tracing.md

### Enable distributed tracing

> Once you've enabled the feature flag, 

Author: ```suggestion
Once you've enabled the feature flag, do the following:
```

> 1. Enable the tracing framework feature by adding the following setting in the `opensearch.yaml` file.

Reviewer: ```suggestion
1. Enable the tracing framework feature by adding the following setting in the `opensearch.yaml` file:
```

> 1. Enable the tracing framework feature by adding the following setting in the `opensearch.yaml` file.

Author: ```suggestion
1. Enable the tracing framework feature by adding the following setting in the `opensearch.yaml` file:
```

> 1. Enable the tracing framework feature by using the following setting, which enables tracing in the opensearch.yaml file

Author: ```suggestion
1. Enable the tracing framework feature by adding the following setting in the `opensearch.yaml` file.
```

```bash
telemetry.feature.tracer.enabled=true
```
{% include copy.html %}

> 2. Then, enable the tracer in the running cluster by adding the following dynamic setting:

Author: ```suggestion
2. Enable the tracer in the running cluster by adding the following dynamic setting:
```

> 2. Then, enable the tracer in the running cluster by adding the following dynamic setting:

Author: ```suggestion
2. Enable the tracer in the running cluster by adding the following dynamic setting:
```

> 2. Also, enable the tracer by using the following dynamic setting that enables tracing in the running cluster:

Author: ```suggestion
2. Then, enable the tracer in the running cluster by adding the following dynamic setting:
```
